apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import { Feature as FeatureEC } from '@pagopa/pagopa-editorial-components';
+import { FeatureProps } from '@pagopa/pagopa-editorial-components/dist/components/Feature/Feature';
+import { FeatureSection } from '@/lib/fetch/types/PageSection';
+
+const makeFeatureProps = ({
+  items,
+  background,
+  ...rest
+}: FeatureSection): FeatureProps => ({
+  items: items.map((item) => ({
+    icon: item.icon,
+    iconColor: item.iconColor,
+    title: item.title,
+    subtitle: item.subtitle,
+    linkText: item.linkText,
+    linkURL: item.linkURL,
+  })),
+  ...(background && { background }),
+  ...rest,
+});
+
+const Feature = (props: FeatureSection) => (
+  <FeatureEC {...makeFeatureProps(props)} />
```

Comment:
I would probably add a `<section/>` surrounding `FeatureEC ` since it's needed for internal links (it's more inside the scope of this PR rather than the style one that will come next).